### ISSUE TITLE
Shortcut the re-provisioning process

### ISSUE THREAD

**Issue by wpscholar** [bradp/vv#146]
One of the frustrations with using VVV is that the more projects you add, the longer it takes to re-provision.  However, if I could just add a site and do a `vagrant reload`, then things would be much faster.    I'm thinking it would be nice to have vv just run the init.sh script, copy over the nginx config files and trigger a `vagrant reload` to add the host entries and restart the services.  Thoughts?

**Comment by bradp** [comment]
This is for sure something I want to add!

**Comment by wpscholar** [comment]
I've added a little file to my VVV config that allows me to do what I described: https://gist.github.com/wpscholar/65e1fa5c571f4be420e6

I just add `sudo bash vvv-provision.sh` to my vvv-init.sh file.  Then, I run `source vvv-init.sh` within VVV. Finally, I run `vagrant reload` to add the domains and restart nginx.  

The only thing I haven't had the time to go and fix is that when I run the command from the `/vagrant/www/{project}` directory (as opposed to the `/srv/www/{project}` directory), it sets the `{vvv_path_to_folder}` to the `/vagrant` path instead of the `/srv` path and it would be great to be able to have it work from either place.
